e2e-tests/portal/cypress/e2e/journeys/login/reset-password.spec.js

```diff
@@ -37,23 +37,29 @@ context('Password management screens', () => {
       resetPassword.submit().click();
 
       cy.url().should('eq', relative('/login?passwordreset=1'));
-      cy.get('[data-cy="password-reset-notification').contains('We\'ve sent an email to the address you have provided. Please check your inbox and spam folder for a message from us. If you require further assistance please contact DigitalService.TradeFinance@ukexportfinance.gov.uk.');
+      cy.get('[data-cy="password-reset-notification').contains(
+        "We've sent an email to the address you have provided. Please check your inbox and spam folder for a message from us. If you require further assistance please contact DigitalService.TradeFinance@ukexportfinance.gov.uk.",
```

Comment:
I've updated this to be in line with our current error handling around passwords. Prior to this, it was not following the pattern -- now we determine this in uniformly